### ISSUE TITLE
React: Visibility Issue with IX Icons on Production Build

### ISSUE THREAD

**Issue by rkoszo** [siemens/ix#1063]
### Prerequisites

- [X] I have not leaked any internal company information.
- [X] I have not posted internal links or code snippets.

### What happened?

We've recently transitioned to IX, but we've encountered a problem. Unfortunately, we are facing a challenge as the IX icons are not visible on our production build.
Interestingly, during my investigation, I found that Font Awesome and MUI icons are displaying correctly. I have been experimenting with various approaches to resolve this issue, but the IX icons remain the same.

Here are the environment details for your reference:
• Operating System: Debian
• Node Versions: 18.16.0 / 20.10.0 / 20.11.0
• React Version: 18.2.0
• @siemens/ix-icons Versions: 2.0.0 / 2.1.0
• @siemens/ix-react Version: 2.0.4
• Bundler: Rollup / SWC

Thank you in advance.

### What type of frontend frameware are you seeing the problem on?

React

### Which version of iX do you use?

2.0.4

### Code to produce this issue.

```shell
I can send our repository or a build in private
```

**Comment by danielleroux** [comment]
Temporary workaround would be to define the icon component before your bootstrap the framework.

e.g

```
import { defineCustomElements } from '@siemens/ix-icons/loader';

defineCustomElements().then(() => {
    const root = createRoot(document.getElementById('app'));
    root.render(<h1>Hello, world</h1>);
})
```

**Comment by matthiashader** [comment]
Internal ref [IX-876]

**Comment by hoeflerjo** [comment]
I also face this issue. 

It is really weird that the issue only arrises for me when using a production build on my Linux system and in no other constellation.
- When using the debug build there is no issue.
- When building it on a Windows system it works on the windows system, but moving it to Linux does not work.

I would assume the build output for node.js projects is completly OS independent. 
Therefore I don't understand why the issue appers on Debian (maybe all Linux distros). It also appers not to be caused by the WebAPI as we tested it on Windows and Linux using Flask, which should also be Platform independent.

**Comment by danielleroux** [comment]
Looks like the issue is solved within latest release v2.2.0 do can confirm this inside your setup?
